arch/risc-v/src/common/espressif/Make.defs

```diff
@@ -146,7 +148,12 @@ endif
 
 chip/$(ESP_HAL_3RDPARTY_REPO):
 	$(Q) echo "Cloning Espressif HAL for 3rd Party Platforms"
-	$(Q) git clone --quiet $(ESP_HAL_3RDPARTY_URL) chip/$(ESP_HAL_3RDPARTY_REPO)
+ifeq ($(STORAGETMP),y)
```

Comment:
could we simply call one function with the enough arguments, which could do all thing internally?